xwiki-platform-core/xwiki-platform-extension/xwiki-platform-extension-index/src/main/java/org/xwiki/extension/index/internal/ExtensionIndexSolrCoreInitializer.java

```diff
@@ -96,6 +96,55 @@ public class ExtensionIndexSolrCoreInitializer extends AbstractSolrCoreInitializ
      */
     public static final String SOLR_FIELD_INDEX_DATE = "s_indexDate";
 
+    /**
+     * The name of the field storing the score of the most critical CVE know for an extension.
+     *
+     * @see #SECURITY_CVE_CVSS for the list of individual CVSS values of the CVEs known for the extension
+     * @since 15.5RC1
+     */
+    public static final String SECURITY_MAX_CVSS = "security_maxCVSS";
+
+    /**
+     * The name of the field storing the IDs of the known CVEs for an extension.
+     *
+     * @since 15.5RC1
+     */
+    public static final String SECURITY_CVE_ID = "security_cveID";
+
+    /**
+     * The link of the CVEs known for an extension.
+     *
+     * @since 15.5RC1
+     */
+    public static final String SECURITY_CVE_LINK = "security_cveLink";
+
+    /**
+     * The values of the individual CVEs known for an extension.
+     *
+     * @see #SECURITY_MAX_CVSS stores the maxium value of this collection
+     */
+    public static final String SECURITY_CVE_CVSS = "security_cveCVSS";
+
+    /**
+     * The number of CVEs know for an extension.
+     */
+    public static final String SECURITY_CVE_COUNT = "security_cveCount";
+
+    /**
+     * The computed version where all the issues are fixed.
+     */
+    public static final String SECURITY_FIX_VERSION = "security_fixVersion";
+
+    /**
+     * A textual explanation of how to upgrade this extension to fix the security issues.
```

Comment:
I'm not sure all advices will be about upgrade, I would put something a bit more generic like "what to do about this security vulnerability".